From 2572cb8dcc40b1458c7dc8a6509583bbc63ce6b5 Mon Sep 17 00:00:00 2001
From: zrj <rimvydas.jasinskas@gmail.com>
Date: Sun, 20 Oct 2019 20:43:47 +0300
Subject: [PATCH 15/39] kernel: Remove not needed header includes in AIO stub.

 AIO not available since 91bcbcd6bef430dfa09526e0af9e24fa8319ae50.
---
 sys/kern/vfs_aio.c | 28 ----------------------------
 1 file changed, 28 deletions(-)

diff --git a/sys/kern/vfs_aio.c b/sys/kern/vfs_aio.c
index e074fb7489..02e7494f89 100644
--- a/sys/kern/vfs_aio.c
+++ b/sys/kern/vfs_aio.c
@@ -21,36 +21,8 @@
  */
 
 #include <sys/param.h>
-#include <sys/systm.h>
-#include <sys/buf.h>
 #include <sys/sysproto.h>
-#include <sys/filedesc.h>
-#include <sys/kernel.h>
-#include <sys/fcntl.h>
-#include <sys/file.h>
-#include <sys/lock.h>
-#include <sys/unistd.h>
-#include <sys/proc.h>
-#include <sys/resourcevar.h>
-#include <sys/signalvar.h>
-#include <sys/protosw.h>
-#include <sys/socketvar.h>
-#include <sys/sysctl.h>
-#include <sys/vnode.h>
-#include <sys/conf.h>
 #include <sys/event.h>
-#include <sys/objcache.h>
-
-#include <vm/vm.h>
-#include <vm/vm_extern.h>
-#include <vm/pmap.h>
-#include <vm/vm_map.h>
-#include <sys/aio.h>
-
-#include <sys/file2.h>
-#include <sys/buf2.h>
-
-#include <machine/limits.h>
 
 int
 sys_aio_return(struct aio_return_args *uap)
-- 
2.23.0

